lib/config/fileconf.go

```diff
@@ -924,6 +928,34 @@ func (r *RestrictedSession) Parse() (*restricted.Config, error) {
 	}, nil
 }
 
+// X11 is a configuration for x11 forwarding
+type X11 struct {
+	// Enabled controls whether x11 forwarding requests can be granted.
+	Enabled string `yaml:"enabled"`
+	// DisplayOffset tells the server what display to start searching from
+	// for an open X11 Server reverse tunnel port (6000 + offset).
+	DisplayOffset *uint `yaml:"display_offset,omitempty"`
+}
+
+// Parse will parse the enhanced session recording configuration.
```

Comment:
1. Fix the godoc - this is not session recording configuration.
2. Please add test coverage showing valid/invalid configurations.